lib/api/v3/users/users_api.rb

```diff
@@ -81,8 +81,6 @@ def user_transition(allowed)
             helpers ::API::V3::Users::UpdateUser
 
             before do
-              authorize_logged_in
```

Comment:
Would a limitation to only show users which are in projects visible to the requesting user also work for the use cases we have? We could also allow admins to see all.

As id on users are sequential, I find it a bit schizophrenic to limit the index action to admins but allow the show action to be called by everybody for every user.